Library/MapleBacon/MapleBacon/MapleBaconConfig.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  MapleBaconConfig.swift
+//  MapleBacon
+//
+//  Created by Danilo Topalovic on 10.03.16.
+//  Copyright © 2016 Zalando SE. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/**
+ * MapleBacon Config
```

Comment:
There's a reason for so few comments in the code, it should be able to speak for itself. This adds no value